views/sections/arge.ejs

```diff
@@ -1,21 +1,25 @@
+<link href="/public/css/share-project-button-style.css" rel="stylesheet">
+
 <div class="row mx-auto">
     <div id="greater-projects-container" class="bg-white col-11 col-xl-10 container-fluid mx-auto">
         <div class="row">
             <div id="projects-container" class="bg-white col-12 px-1 px-sm-2 px-lg-4 mx-auto">
-                <div class="row mx-auto px-1 px-sm-1 py-2 my-4">
+                <div class="row mx-auto px-1 px-sm-1 py-2 my-4 justify-content-evenly">
                     <div class="mt-2 text-center p-0">
                         <h2>Projelerimiz</h2>
                     </div>
                     <% for (const [index, project] of projects.entries()) { %>
-                        <div class="col-6 col-md-4 col-xl-3 my-1 my-sm-3 px-1 px-md-2 px-xl-4 py-xl-2">
+                        <div class="col-6 col-md-4 col-xl-3 my-1 my-sm-3 px-1 px-md-2 px-xl-4 py-xl-2 text-center">
                             <div class="card rounded h-100 event-card-inner shadow bg-light" data-bs-toggle="modal"
                                 data-bs-target="#exampleProjectModal<%=index%>">
                                 <div class="card-body d-flex flex-column p-3">
                                     <img src="<%= project.img %> >" alt="" class="img-fluid d-block mx-auto mb-3">
                                     <h4 class="mb-3">
                                         <%= project.name %>
                                     </h4>
-                                    <button class="btn events-color mt-auto border-0">Detaylar &gt;</button>
+                                    <button class="btn shadow border-1 button-element-event-special" onclick="location.href='/idea'">
```

Comment:
The `onclick` attribute here is redundant and should be removed, since the clicks are handled by the parent div. You probably accidentally overlooked that :)